README.md

```diff
@@ -121,19 +123,20 @@ This is deployed using Hokusai to manage Docker and Kubernetes. To replicate thi
 
 ### Testing
 
-- Run tests in the Docker Compose test stack via Hokusai:
+* Run tests in the Docker Compose test stack via Hokusai:
 
   ```
   $ hokusai test
   ```
 
-- Or, to run tests locally: `npm test` to run the entire suite `npm run watch`
+* Or, to run tests locally: `npm test` to run the entire suite `npm run watch`
   to spin up the test watcher
 
 ### Deployment
 
 PRs merged to the `master` branch are automatically deployed to staging. The
-release on staging can be promoted to production via the command `hokusai pipeline promote`. See Hokusai's
+release on staging can be promoted to production via the command `hokusai
+pipeline promote --git-remote origin`. See Hokusai's
```

Comment:
I added this change

Reply:
I think I was wrong and this should actually refer to the `upstream` or `artsy` remote. (Usually it _is_ `origin` within Circle scripts though.)

In any case, please try it before merging in case I've missed other details.